pkg/server/container_create_windows.go

```diff
@@ -318,11 +319,46 @@ func (c *criService) generateContainerSpec(id string, sandboxID string, sandboxP
 		}
 		if strings.HasPrefix(formattedSource, `\\.\PHYSICALDRIVE`) {
 			mo.Type = "physical-disk"
+			if sandboxPlatform == "linux/amd64" {
+				mo.Options = append(mo.Options, "bind")
+			}
 		} else if strings.HasPrefix(formattedSource, `\\.\pipe`) {
 			// mo.Type == "" for pipe but we don't want to Stat the path.
 			if sandboxPlatform == "linux/amd64" {
 				return nil, errors.Errorf(`pipe mount.HostPath '%s' not supported for LCOW`, formattedSource)
 			}
+		} else if strings.HasPrefix(formattedSource, "automanage-vhd://") {
+			formattedSource = strings.TrimPrefix(formattedSource, "automanage-vhd://")
+			s, err := c.os.Stat(formattedSource)
+			if err != nil {
+				return nil, errors.Wrapf(err, "failed to Stat automanage-vhd:// mount.HostPath %q", formattedSource)
+			}
+			if s.IsDir() {
+				// TODO: JTERRY75 - This is a hack and needs to be removed. The
+				// orchestrator should be controlling the entry host path. For
+				// now if its a directory copy the template vhd and update the
+				// source.
+				if c.config.PluginConfig.TmpTemplateVhdPath == "" {
```

Comment:
As this is generalized beyond just the /tmp case, we should rename the config value to something like `AutoManageVHDTemplatePath`.

Reply:
Deal